source/templates.tex

```diff
@@ -4386,8 +4386,8 @@
 
 \pnum
 A \grammarterm{qualified-id}
-whose terminal name is dependent and that is in a type-only context
-is considered to denote a type.
+whose terminal name is dependent and in a type-only context
+is interpreted as a \grammarterm{simple-type-specifier}.
```

Comment:
If it is not in a type-only context, how to interpret it? Is it just an expression or Is it determined by the result of name lookup in the instantiation context?

Reply:
Fixed